lib/iris/tests/unit/fileformats/grib/message/test_GribMessage.py

```diff
@@ -91,46 +91,21 @@ def test_no_bitmap(self):
         self.assertEqual(result.shape, self.shape)
         self.assertArrayEqual(result, expected)
 
-    @tests.skip_dask_mask
-    def test_bitmap_present_int_data(self):
+    def test_bitmap_present(self):
```

Comment:
I suppose it wouldn't hurt to have them both, but I believe this test was split in two when we had the "nan array for masks" behaviour, which differed depending on the dtype. Now that we don't have that it seemed sensible just to collapse them back into a single test again.